stdlib/kafka/to_test.go

```diff
@@ -548,77 +549,51 @@ func TestToKafka_Process(t *testing.T) {
 }
 
 func TestToKafka_NewTransformation(t *testing.T) {
-	testCases := []struct {
```

Comment:
There is basically no change to the test cases themselves. I just made the test cases to use the new harness.
The reviewer may check https://github.com/influxdata/flux/blob/f3e476b1ff1941ee636d53ec4506451a63b9c3ec/stdlib/kafka/to_test.go#L552 instead of trying hard to look through the diff.